src/components/vouchers/VoucherRedeem.tsx

```diff
@@ -0,0 +1,298 @@
+
+                <Wrapper style={styles.noPaddingTop}>
+                  <RegularText lightest>{texts.voucher.detailScreen.redeemDescription}</RegularText>
+                </Wrapper>
+
+                <Wrapper style={styles.progressContainer}>
+                  <RegularText style={styles.progressOvertitle} lightest>
+                    {texts.voucher.detailScreen.progressTitle}
+                  </RegularText>
+
+                  <CircularProgress
+                    activeStrokeColor={colors.primary}
+                    inActiveStrokeColor={colors.lighterPrimary}
+                    maxValue={100}
+                    radius={120}
+                    showProgressValue={false}
+                    subtitle={texts.voucher.detailScreen.progressSubtitle}
+                    subtitleColor={colors.surface}
+                    subtitleFontSize={normalize(15)}
+                    title={`${String(minutes).padStart(2, '0')}:${String(seconds).padStart(
+                      2,
+                      '0'
+                    )}`}
+                    titleColor={colors.surface}
+                    titleFontSize={normalize(50)}
+                    titleStyle={styles.progressTitle}
+                    value={((defaultTime - remainingTime) / defaultTime) * 100}
+                  />
+                </Wrapper>
+
+                <Wrapper>
+                  <Touchable
+                    onPress={() => {
+                      setIsVisible(false);
+                      setIsRedeemVoucher(false);
+                      setIsChecked(false);
+                    }}
+                  >
+                    <BoldText small center underline lightest>
+                      {texts.voucher.detailScreen.close}
+                    </BoldText>
+                  </Touchable>
+                </Wrapper>
+              </>
+            ) : (
+              <>
+                <Wrapper>
+                  <BoldText lightest>{texts.voucher.detailScreen.sheetTitle}</BoldText>
+                </Wrapper>
+
+                <Wrapper style={styles.noPaddingTop}>
+                  <RegularText lightest>{texts.voucher.detailScreen.sheetDescription}</RegularText>
+                </Wrapper>
+
+                <Wrapper style={styles.noPaddingTop}>
+                  <Checkbox
+                    checked={isChecked}
+                    checkedColor={colors.surface}
+                    checkedIcon="check-square-o"
+                    containerStyle={styles.checkbox}
+                    title={texts.voucher.detailScreen.checkboxLabel}
+                    onPress={() => setIsChecked(!isChecked)}
+                    uncheckedColor={colors.surface}
+                    uncheckedIcon="square-o"
+                    lightest
+                  />
+                </Wrapper>
+
+                {!!maxPerPerson && maxPerPerson > 1 && (
+                  <Wrapper style={styles.noPaddingTop}>
+                    <View style={styles.quantityContainer}>
```

Comment:
`WrapperRow` could be used here

Reply:
But `WrapperRow` does not have `alignItems` prop. Therefore I have to add the style prop. Or I can add `alignItems="center"` prop to `WrapperRow`.

Reply:
I added a style and changed the `View` to `WrapperRow`.